hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/search/search.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Input, Select } from 'antd';
+import { Option } from '@/v2/components/select/singleSelect';
+
+type SearchProps = {
+  searchColumn?: string;
+  searchOptions?: Option[];
+  onSearch: (
+    arg0: string,
```

Comment:
This new search icon added will implement searching from the UI itself right and wont be requiring anything from the backend I hope?
<img width="1658" alt="image" src="https://github.com/user-attachments/assets/e62609e3-2789-4c28-b6c6-9d4b9d8f8879">

Reply:
Yes, there won't be any backend changes required.
Previously we had a search icon beside every column which was searchable.
<img width="444" alt="Screenshot 2024-08-16 at 13 00 16" src="https://github.com/user-attachments/assets/f9643036-2b3e-42f6-967a-17ff5deb3aec">
But this would take up column space and was un-intuitive.
Hence we moved to a separate search where we can utilise the existing empty space in the heading to provide this search.